code/game/objects/items/storage/book.dm

```diff
@@ -198,7 +198,68 @@ GLOBAL_LIST_INIT(bibleitemstates, list("bible", "koran", "scrapbook", "burning",
 			new /obj/item/nullrod/claymore(get_turf(sword))
 			user.visible_message("<span class='notice'>[user] has purified the [sword]!</span>")
 			qdel(sword)
-
+	//yogs start -- More Cult conversions!//
+	else if(istype(A, /obj/item/melee/cultblade) && !iscultist(user)) 
```

Comment:
Can you shunt this long else-if chain into a proc of ``/obj/item/storage/book/bible``? Like
```dm
	//yogs start -- other conversions
	else if(other_conversions())
		//a null if-statement with all the action happening in the proc
	//yogs end
	else if(istype(A, /obj/item/soulstone) && !iscultist(user))
```
Or something?